lib/locales/en-KE.yml

```
@@ -0,0 +1,281 @@
+en-KE:
+  faker:
+    internet:
+      domain_suffix:
+        - ke
+        - com
+        - co.ke
+        - org.ke
+        - or.ke
+        - net
+        - go.ke
+        - biz
+        - ac.ke
+        - info.ke
+    address:
+      default_country:
+        - Republic of Kenya
+        - KE
```

```suggestion
      default_country:
        - Republic of Kenya
```

I believe default_country is supposed to be a single entry here. Is `Republic of Kenya`? what you intended?